API/common/paths.py

```diff
@@ -56,16 +56,10 @@
 #
 
 # Path to the iotjs-test-results folder.
-IOTJS_WEB_PATH = os.path.join(PROJECTS_ROOT, 'iotjs-test-results')
-
-# Path to the data folder within iotjs-test-results.
-IOTJS_WEB_DATA_PATH = os.path.join(IOTJS_WEB_PATH, 'data')
+IOTJS_TEST_RESULTS_WEB_PATH = os.path.join(PROJECTS_ROOT, 'iotjs-test-results')
```

Comment:
I think this should be `IOTJS_TEST_RESULTS_WEB_PATH = os.path.join(PROJECTS_ROOT, 'js-remote-test')

Reply:
It's tricky.  In the project folder we need the `gh-pages` branch of the `js-remote-test` (at the current status of this project), which only contains the iotjs test results. But, it is a good notice, because the `init.sh` contains the wrong url. I've fixed this there.